atlite/resources/windturbine/2016CACost_NREL_Reference_10MW_205_offshore.yaml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: 2022 The Atlite Authors
+#
+# SPDX-License-Identifier: CC-BY-4.0
+
+# Turbine characteristic data
+name: 2016CACost_NREL_Reference_10MW_205
```

Comment:
We should use a consistent file naming scheme for the files and turbines:

Until now we are using

`<manufacturer>_<manufacturer turbine name>_<power in MW>.yaml`

Could you adjust the file names accordingly? For the NREL turbines it might make sense to e.g. have `NREL_ReferenceTurbine_ATB2020_10MW.yaml`

The `name: ` value inside the `.yaml` files can stay like the original used in NREL's GitHub repo.